src/main/kotlin/it/pagopa/wallet/domain/details/CardDetails.kt

```diff
@@ -4,11 +4,12 @@ import it.pagopa.generated.wallet.model.WalletCardDetailsDto
 
 /** Data class that maps WalletDetails for CARD instrument type */
 data class CardDetails(
+    val type: String,
     val bin: String,
     val maskedPan: String,
     val expiryDate: String,
```

Comment:
I think here we could use different class so that we don't mix up strings, wdyt?

Reply:
It's a good point